beancount_reds_importers/libreader/ofxreader.py

```diff
@@ -56,6 +56,8 @@ def get_balance_positions(self):
             yield pos
 
     def get_available_cash(self):
+        if not hasattr(self.ofx_account.statement, 'available_cash'):
+            return None
         return self.ofx_account.statement.available_cash
```

Comment:
return getattr(self.ofx_account.statement, 'available_cash', None)  # pythonic

Reply:
That's better. Thanks. Done.